src/main/java/com/dev/cinema/service/implementation/ShoppingCartServiceImpl.java

```diff
@@ -53,7 +54,7 @@ public void addSession(MovieSession movieSession, User user) {
 
     @Override
     public void clear(ShoppingCart shoppingCart) {
-        shoppingCart.getTickets().clear();
+        shoppingCart.setTickets(Collections.emptyList());
```

Comment:
Collections.emptyList() will return immutable list. Is this okay for our purpose?

Reply:
Yes, it works. Tested

Reply:
That's okay cause later you are fetching it from DB